src/pages/atomicdex/api/v20-dev/non_fungible_tokens/index.mdx

```diff
@@ -0,0 +1,577 @@
+    ],
+    "url": "https://moralis-proxy.komodo.earth"
+  }
+}
+```
+</CodeGroup>
+
+
+### Response Parameters
+
+| Parameter            | Type             | Description                                                                                      |
+| ---------------------| ---------------- | ------------------------------------------------------------------------------------------------ |
+| amount               | string           | Amount of tokens transferred.                                                                    |
+| block_hash           | string           | Hash of block in which transfer occurred.                                                        |
+| block_number         | integer          | Height of block in which transfer occurred.                                                      |
+| block_timestamp      | string           | Block date/time string. CONFIRM: SHOULD THIS NOT BE AN EPOCH TIMESTAMP?                          |
+| chain                | string           | Chain name. One of `AVALANCHE`, `BSC`, `ETH`, `FANTOM`, or `POLYGON`.                            |
+| collection_name      | string           | May be `null`. Name of collection which token belongs to.                                        |
+| contract_type        | string           | Contract type. `ERC721` or `ERC1155`.                                                            |
+| token_address        | string           | Address of token transferred.                                                                    |
+| token_id             | string           | Token ID.                                                                                        |
+| possible_spam        | boolean          | Possible spam.  MORE  INFO?                                                                      |
+| transaction_hash     | string           | Transaction ID of the transfer.                                                                  |
+| transaction_index    | string           | Transaction index of the transfer.                                                               |
+| log_index            | string           | Log index of the transfer.                                                                       |
+| value                | string           | Tranfer value.                                                                                   |
+| transaction_type     | string           | Transation type. `Single` or `WHAT IS THE OTHER TYPE?`.                                          |
+| from_address         | string           | Address of previous owner which sent the token(s).                                               |
+| to_address           | string           | Address of new owner which received the token(s).                                                |
+| status               | string           | Transfer status. WHAT ARE THE POSSIBLE VALUES?                                                   |
```

Comment:
Receive or Send

```rust
pub(crate) enum TransferStatus {
    Receive,
    Send,
}
```

It basically shows that if from_address was the my_address (weans our user address), then user sent nft,
If to_address == user address, then user received nft in transaction.
It was made for GUI team and also it enable user not to check from or to address with their wallet address to understand in this transaction they received or sent nft.

```rust
let status = if wallet_address.to_lowercase() == transfer_wrapper.to_address {
                        TransferStatus::Receive
                    } else {
                        TransferStatus::Send
                    };
```

Reply:
thanks, updated description